### PR TITLE
Added Greek language file

### PR DESCRIPTION
**Pull request by dimitrios-iliadis**
*Fixes #*

**Describe what the proposed change does**
-
-
-

### CODE REVIEW

**Comment by TheCycoONE** [comment]
Could you add "greek" to the list of language files in .luacheckrc line 113. If not, may I?

**Comment by dimitrios-iliadis** [comment]
> Could you add "greek" to the list of language files in .luacheckrc line 113. If not, may I?

I have added Greek. Can you check that everything is okay now?

File: CorsixTH/Lua/languages/greek.lua [hunk 1]
```diff
@@ -0,0 +1,3349 @@
...
+  [20] = "  20%  ",
+  [30] = "  30%  ",
+  [10] = "  10%  ",
+}
+newspaper = {
+  [1] = {
+    [1] = "ΓΙΑΤΡΟΣ ΔΗΜΙΟΥΡΓΕΙ ΚΟΙΝΩΝΙΚΗ ΚΑΤΑΚΡΑΥΓΗ",
+    [2] = "ΤΡΕΛΟΣ ΓΙΑΤΡΟΣ ΠΑΡΙΣΤΑΝΕΙ ΤΟ ΘΕΟ",
+    [3] = "ΣΟΚ ΦΑΡΣΕΡ-ΣΤΑΪΝ",
+    [4] = "WHAT WAS ON LAB SLAB?",
```
Comment (lewri) [inline CorsixTH/Lua/languages/greek.lua:469]: This one missed translation I'm guessing, or perhaps there wasn't an exact translation available?

File: CorsixTH/Lua/languages/greek.lua [hunk 2]
```diff
@@ -0,0 +1,3349 @@
...
+    [5] = "ΕΠΙΔΡΟΜΗ ΒΑΖΕΙ ΤΕΛΟΣ ΣΕ ΕΠΙΚΙΝΔΥΝΗ ΕΡΕΥΝΑ",
+  },
+  [2] = {
+    [1] = "DR-UNK AS A SKUNK",
```
Comment (lewri) [inline CorsixTH/Lua/languages/greek.lua:473]: This one might've also been missed. But I suppose you could play with words to achieve a newsapaper title of similar effect.

File: CorsixTH/Lua/languages/greek.lua [hunk 3]
```diff
@@ -0,0 +1,3349 @@
...
+  },
+  deaths = "Θάνατοι",
+  money_out = "Έξοδα",
+}
+staff_list = {
+  skill = "ΔΕΞΙΟΤΗΤΑ",
+  total_wages = "ΣΥΝΟΛΙΚΟΙ ΜΙΣΘΟΙ",
+  tiredness = "ΚΟΥΡΑΣΗ",
+  morale = "ΗΘΙΚΟ",
+}
```
Comment (lewri) [inline CorsixTH/Lua/languages/greek.lua:3349]: Need to add an empty line at the bottom here (no spaces etc.)

File: CorsixTH/Lua/languages/greek.lua [hunk 4]
```diff
@@ -0,0 +1,3349 @@
...
+    [1] = "ΓΙΑΤΡΟΣ ΜΕ ΡΑΜΜΑΤΑ!",
+    [2] = "SLACK QUACK",
+    [3] = "ΕΝΟΧΟΠΟΙΗΤΙΚΗ ΔΙΑΓΝΩΣΗ",
+    [4] = "ΑΤΣΟΥΜΠΑΛΟΣ ΣΥΜΒΟΥΛΟΣ",
+  },
+  [7] = {
+    [1] = "ΓΙΑΤΡΟΣ ΔΕΝ ΕΧΕΙ ΤΙΠΟΤΑ ΝΑ ΚΑΝΕΙ",
+    [2] = "ΧΕΙΡΟΥΡΓΟΣ 'ΕΓΧΕΙΡΙΖΕΙ' ΤΟΝ ΕΑΥΤΟ ΤΟΥ",
+    [3] = "BOG BISHOP-BASHING",
+    [4] = "DOCTOR'S HANDLE SCANDAL",
```
Comment (lewri) [inline CorsixTH/Lua/languages/greek.lua:510]: Word play again here may hinder translation I guess.

**Review by lewri — COMMENTED** [review]
Obviously we can't vouch for translations by the community but looks nicely thorough.

The few strings still in English I flagged may be fine as is, but feel free to translate it with a little artistic merit if you like to make the puns work in your own language if you want.

**Comment by TheCycoONE** [comment]
Thanks @dimitrios-iliadis

**Comment by TheCycoONE** [comment]
@dimitrios-iliadis because you did this development on your master branch you may hit problems if you try to do anything else.

I recommend `git fetch origin; git reset --hard origin/master` - in the future you will want to look into starting a branch to make changes in.

**Pull request merged**